.yamllint

```diff
@@ -0,0 +1,7 @@
+extends: default
+
+rules:
+  truthy: disable
+  line-length:
+    max: 1500
+  trailing-spaces: disable
```

Comment:
This is to ignore the `trailing-spaces` error

Reply:
Trailing spaces seems like something we should flag?

Reply:
The yamllint configuration will cause it to ignore spaces at the end of lines. Can we have it?

Reply:
Right, I'm saying: I think we should lint that there are no trailing spaces (unless it is prohibitive for some reason).

Reply:
excluded it from this config file